build/tools/releaser-post.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# This script will automate the file alteration, which involves
+# 1. Updating version of the CLI in version/version.go
+# 2. Updating version of noobaa core image in pkg/options.go => ContainerImageTag
+# 3. Run make gen
+# 4. Commit the changes with automated commit message
+
+set -eu
+set -o pipefail
+
+dir=$(dirname "$0")
+
+# Source the utilities
+source $dir/utils.sh
+
+version=$(bump_semver_patch $(get_noobaa_version))
+DRY_RUN=${DRY_RUN:="false"}
+
+# Update version of the CLI in version/version.go
+function update_version() {
+	local version_file=version/version.go
+	# Replace version line with new version using perl because sed is not compatible with both mac and linux
+	# and awk solution is clumsy
```

Comment:
```suggestion
	# Replace the version line with the new version using perl because sed is not compatible with both mac and linux
	# and awk solution is clumsy
```